sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -85,22 +85,29 @@ internal class RadarApiClient(
         )
     }
 
-    private fun parseMetaAndConfigure(res: JSONObject?) {
+    data class RadarMeta(
+        val config: JSONObject?,
+        val remoteTrackingOptions: RadarTrackingOptions?
+    )
+
+    private fun parseMeta(res: JSONObject?): RadarMeta {
         var meta: JSONObject? = null
         if (res?.has("meta") == true) {
             meta = res.getJSONObject("meta")
         }
 
+        var config: JSONObject? = null
         if (meta?.has("config") == true) {
-            val config = meta.getJSONObject("config")
-            RadarSettings.setConfig(context, config)
+            config = meta.getJSONObject("config")
         }
 
+        var remoteTrackingOptions: RadarTrackingOptions? = null
         if (meta?.has("trackingOptions") == true) {
```

Comment:
we don't use an explicit boolean; we just return trackingOptions in the meta if enabled, and don't if not.